src/home_robot/home_robot/motion/stretch.py

```diff
@@ -67,7 +67,7 @@
     ]
 )
 PIN_CONTROLLED_JOINTS = [
-    # "base_x_joint",
+    "base_x_joint",
```

Comment:
@exhaustin can you confirm this looks right?

Reply:
Yep, confirmed by calling `manip_fk()` with this configuration yesterday and keyframes looked as expected.

Reply:
I will be using this for my manipulation planning. The video I posted yesterday uses pinocchio with this IK solver to execute the sequence of actions. My use-case is a bit more generic than Sam's I think where the initial pose of the robot shouldn't matter. If it does though, I can always comment this out + fix the base in URDF and use that for my 1st milestone.